scripts/jenkins/gitlint.ini

```diff
@@ -0,0 +1,20 @@
+[general]
+verbosity = 2
+
+[title-max-length]
+line-length=80
+
+[title-must-not-contain-word]
+words=wip,WIP
+
+[title-match-regex]
+regex=^SCRD-.*|.*\((bsc#[1-9]+\s?)+\)$
```

Comment:
I think that we can add trello cards id too

Reply:
How does a trello card id look like?

Reply:
The trello card id is an alphanumeric string, you can find a sed script that turns references to bugs into links to the bugs at http://git.suse.provo.cloud/cgit/hp/config/tree/modules/hp/files/git/commit-filter.sh?h=suse/gozer for reference.